incubator/knative/templates/eventing/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.eventing.install -}}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: eventing-controller
+  namespace: {{ .Values.eventing.namespace }}
```

Comment:
afaik the app/chart/rel/heritage labels should be at this level

Reply:
We addressed this in the latest commit